src/app/pages/model-3d/model-3d-list/model-3d-list.page.ts

```diff
@@ -0,0 +1,76 @@
+import {
+  Component,
+  OnInit,
+  inject,
+  ChangeDetectionStrategy,
+  effect,
+} from '@angular/core';
+import {
+  FormBuilder,
+  FormGroup,
+  Validators,
+  ReactiveFormsModule,
+} from '@angular/forms';
+import { CommonModule } from '@angular/common';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatInputModule } from '@angular/material/input';
+import { MatSelectModule } from '@angular/material/select';
+import { MatButtonModule } from '@angular/material/button';
+import { MatIcon } from '@angular/material/icon';
+import { TranslatePipe } from '@ngx-translate/core';
+import { GeneralContainerComponent } from '@components/layout';
+import { GlbViewerComponent } from '@components/model3D';
+import { AlertTypeEnum } from '@common/enums';
+import { AlertService, FormsService } from '@services/general';
+import { PagesUrlsEnum } from '@common/enums';
+import { Model3DCreateHttpService } from '@services/http';
+import { Router } from '@angular/router';
+
+
+@Component({
+  selector: 'app-model-3d-list',
+  templateUrl: './model-3d-list.page.html',
+  styleUrls: ['./model-3d-list.page.scss'],
+  standalone: true,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  imports: [
+    CommonModule,
+    ReactiveFormsModule,
+    MatFormFieldModule,
+    MatInputModule,
+    MatSelectModule,
+    MatButtonModule,
+    GeneralContainerComponent,
+    GlbViewerComponent,
+    MatIcon,
+  ],
+})
+export class Model3DListPage implements OnInit {
+  form: FormGroup;
+  animals: { id: number; name: string }[] = [];
```

Comment:
La misma historia de la interfaz.

De hecho ahora como se repite esto en dos lugares distintos, es ya una alerta de que hay que hacer una interfaz y referenciar la misma.


```suggestion
  animals: MiInterfazQueDefineElTipo[] = [];
```